specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/cadl/multivariate/models.cadl

```diff
@@ -0,0 +1,370 @@
+  status?: ModelStatus;
+
+  @doc("Error messages when failed to create a model.")
+  errors?: ErrorResponse[];
+
+  @doc("Diagnostics information to help inspect the states of model or variable.")
+  diagnosticsInfo?: DiagnosticsInfo;
+}
+
+@doc("An optional field, indicating the manner to align multiple variables.")
+model AlignPolicy {
+  @doc("""
+An optional field, indicating how to align different variables to the same
+time-range. Either Inner or Outer. 
+""")
+  alignMode?: AlignMode;
+
+  @doc("""
+An optional field, indicating how missing values will be filled. One of
+Previous, Subsequent, Linear, Zero, Fixed.
+""")
+  fillNAMethod?: FillNAMethod;
+
+  @doc("An optional field. Required when fillNAMethod is Fixed.")
+  paddingValue?: float32;
+}
+
+@doc("Diagnostics information to help inspect the states of model or variable.")
+model DiagnosticsInfo {
+  modelState?: ModelState;
+  variableStates?: VariableState[];
+}
+
+model ModelState {
+  @doc("""
+This indicates the number of passes of the entire training dataset the
+algorithm has completed.
+""")
+  epochIds?: int32[];
+
+  @doc("""
+List of metrics used to assess how the model fits the training data for each
+epoch.
+""")
+  trainLosses?: float32[];
+
+  @doc("""
+List of metrics used to assess how the model fits the validation set for each
+epoch.
+""")
+  validationLosses?: float32[];
+
+  @doc("Latency for each epoch. ")
+  latenciesInSeconds?: float32[];
+}
+
+@doc("Response of getting a model.")
+@resource("multivariate/models")
+model AnomalyDetectionModel {
+  @doc("Model identifier.")
+  @key
+  @format("uuid")
+  modelId: string;
+
+  @doc("Date and time (UTC) when the model was created.")
+  createdTime: zonedDateTime;
+
+  @doc("Date and time (UTC) when the model was last updated.")
+  lastUpdatedTime: zonedDateTime;
+
+  @doc("""
+Training result of a model including its status, errors and diagnostics
+information.
+""")
+  modelInfo?: ModelInfo;
+}
+
+@doc("Response of listing models.")
+@pagedResult
+model ModelList {
+  @doc("List of models.")
+  @items
+  models: AnomalyDetectionModel[];
+
+  @doc("Number of trained multivariate models.")
+  currentCount: int32;
+
+  @doc("Maximum number of models that can be trained for this Anomaly Detector resource.")
+  maxCount: int32;
+
+  @doc("The link to fetch more models.")
+  @nextLink
+  nextLink?: string;
+}
+
+model MultivariateLastDetectionOptions {
+  @doc("""
+This contains the inference data, including the name, timestamps(ISO 8601) and
+values of variables.
+""")
+  variables: VariableValues[];
+
+  @doc("""
+An optional field, which is used to specify the number of top contributed
+variables for one anomalous timestamp in the response. The default number is
+10.
+""")
+  topContributorCount: int32;
+}
+
+model VariableValues {
+  @doc("Variable name of last detection request.")
+  variable: string;
+
+  @doc("Timestamps of last detection request")
+  timestamps: string[];
```

Comment:
Java apiview: To Service: what format is it? RFC1123?

Reply:
Then I think it should be "zonedDateTime"? @catalinaperalta

Reply:
zonedDateTime says it's for this format: "April 10th at 3:00am in PST", which isnt ISO 8601. It should remain as string in this case.

Reply:
@catalinaperalta Do you have the source on this?

I think it is just swagger "date-time", which is [RFC 3339](https://tools.ietf.org/html/rfc3339#section-5.6)

https://cadlplayground.z22.web.core.windows.net/cadl-azure/?c=aW1wb3J0ICJAY2FkbC1sYW5nL3Jlc3QiOwoKdXNpbmcgQ2FkbC5IdHRwOwoKQHNlcnZpY2UoewogIHRpdGxlOiAiQnVpbHRpbiIsCiAgdmVyc2lvbjogIjEuMC4wIiwKfSkKbmFtZXNwYWNlxkzHMDsKCm1vZGVsIMcQIMRUZHVyYXTFQcgKO8QWYXRlVGltZTogem9uZWRExw87Cn0KCkByb3V0ZSgiL2LGRyIpCmludGVyZsRwx1tPcMVdcmVhZCgpOsUMICBAYm9keSDEBTrIKMRsfcRW

https://swagger.io/docs/specification/data-models/data-types/

Anyway this is not a string.